### PR TITLE
Vendor dependencies, update k8s to 1.16.3

### PR DESCRIPTION
**Pull request by tomahawk28**
Towards giantswarm/giantswarm#7675

### CODE REVIEW

File: Gopkg.lock [hunk 1]
```diff
@@ -1291,35 +854,30 @@
   version = "kubernetes-1.15.4"
```
Comment (sslavic) [inline Gopkg.lock]: @tomahawk28 this was not updated

File: Gopkg.lock [hunk 2]
```diff
@@ -1291,35 +854,30 @@
...
     "pkg/apis/apiextensions",
     "pkg/apis/apiextensions/v1beta1",
     "pkg/client/clientset/clientset",
     "pkg/client/clientset/clientset/scheme",
     "pkg/client/clientset/clientset/typed/apiextensions/v1beta1",
-    "pkg/features",
   ]
   pruneopts = "UT"
   revision = "3de75813f60447b64b34060fdaabd9bc23627eba"
   version = "kubernetes-1.15.4"
```
Comment (sslavic) [inline Gopkg.lock]: @tomahawk28 this as well, and all the other k8s.io ones

**Comment by tomahawk28** [comment]
Tested on `geckon`

**Comment by tomahawk28** [comment]
Called with @rossf7. We agreed on refactoring on `project` usage by next PR.

**Comment by tomahawk28** [comment]
Going to ping

**Pull request merged**